src/main/java/org/jenkinsci/remoting/engine/EngineUtil.java

```diff
@@ -91,4 +96,24 @@ protected static Properties readResponseHeaders(InputStream inputStream) throws
             response.put(line.substring(0,idx).trim(), line.substring(idx+1).trim());
         }
     }
+
+    /**
+      * Closes the item and logs error to the log in the case of error.
+      * Logging will be performed on the {@code WARNING} level.
+      * @param toClose Item to close. Nothing will happen if it is {@code null}
+      * @param logger Logger, which receives the error
+      * @param closeableName Name of the closeable item
+      * @param closeableOwner String representation of the closeable holder
+      */
+    static void closeAndLogFailures(@CheckForNull Closeable toClose, @Nonnull Logger logger,
```

Comment:
:ant:; this code idiom was never a good idea in Java 6- IMO (better to just let any hypothetical exception from `close` be thrown up the stack), and is completely unnecessary in Java 7+, since `try`-with-resources calls `addSuppressed` when necessary. So delete this unused method, and when fixing FB warnings, just use the modern `try` style and you are fine.